hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/TestFSConfigToCSConfigConverter.java

```diff
@@ -172,37 +170,41 @@ private void createConverter() {
   public void testDefaultMaxAMShare() throws Exception {
     converter.convert(config);
 
-    Configuration conf = converter.getCapacitySchedulerConfig();
+    CapacitySchedulerConfiguration conf = converter.getCapacitySchedulerConfig();
     String maxAmShare =
         conf.get(CapacitySchedulerConfiguration.
             MAXIMUM_APPLICATION_MASTERS_RESOURCE_PERCENT);
```

Comment:
There is a getMaximumApplicationMasterResourcePercent() method in the CapacitySchedulerConfiuration class which returns this value. Could you please use that here?